chainList.json

```diff
@@ -1,40 +1,4 @@
 [
-  {
```

Comment:
Was it intentional to make changes / reorder this file?

Reply:
The order was not consistently applied, so I updated the file to list all chains by name.